packages/thirdweb/src/react/providers/thirdweb-provider.tsx

```diff
@@ -180,7 +169,7 @@ export type ThirdwebProviderProps = {
    * - [rainbowConfig](https://portal.thirdweb.com/references/typescript/v5/rainbowConfig)
    * - [zerionConfig](https://portal.thirdweb.com/references/typescript/v5/zerionConfig)
    */
-  wallets?: WalletConfig[];
+  wallets: WalletConfig[];
```

Comment:
how come this is mandatory now?

Reply:
Mandatory in `react` which no-one should import from. We pass defaults from React/RN